### PR TITLE
Add more rounded version of star

### PR DESCRIPTION
**Pull request by sinout**
A star icon which should be used for feedback changes (https://meetup.atlassian.net/wiki/spaces/elevate/pages/679708686/MUP+entrypoint+new+question+v1). I don't know whether the naming of new icon suits.

### CODE REVIEW

**Comment by mmcgahan** [comment]
I think something has gone wrong here - there are 578 files in the diff, most of which are unrelated to the star icon. I haven't made icons for this repo, but I would expect a diff that looks more like #100

**Comment by sinout** [comment]
I've also never made icons, just followed the steps in Readme. It is a little strange cause in this PR [#107](https://github.com/meetup/swarm-icons/pull/107) there are also ~500 files in diff.

**Comment by mmcgahan** [comment]
Okay then, works for me! If you could bump the minor version here: https://github.com/meetup/swarm-icons/blob/master/Makefile#L2

and add a note in the CHANGELOG about what was added, you should be good to go

**Comment by sinout** [comment]
@mmcgahan No problems. Done.

**Review by eilinora — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# [3.7]
+
+- **Change** Added new star icon with more rounded edges (`roudned-star`)
```
Comment (jaclynj) [inline CHANGELOG.md]: `roudned-star` - typo! rounded is spelled wrong but it looks like this should be `star-rounded` anyway

Comment (osermay) [reply CHANGELOG.md]: `star--rounded` does seem correct

Comment (sinout) [reply CHANGELOG.md]: The sketch file is called `star-rounded`. I don't know why after running `yarn run build` it created an icon called `rounded-star`.

Comment (sinout) [reply CHANGELOG.md]: @osermay Then I guess we will have `star--rounded--small` for small variant.

Comment (sinout) [reply CHANGELOG.md]: Okay. I get this. I've downloaded and installed Sketch and renamed the artboards inside Sketch app.

**Pull request merged**